neuromation/cli/admin.py

```diff
@@ -14,6 +15,35 @@ def admin() -> None:
     """Cluster administration commands."""
 
 
+@command()
+@async_cmd()
+async def get_clusters(root: Root) -> None:
+    """
+    Print the list of available clusters.
+    """
+    fmt = ClustersFormatter()
+    clusters = await root.client._admin.list_clusters()
+    pager_maybe(
+        fmt(clusters.values()), root.tty, root.terminal_size,
+    )
+
+
+@command()
+@click.argument("cluster_name", required=True, type=str)
+@click.argument("config", required=True, type=click.File(encoding="utf8", lazy=False))
+@async_cmd()
+async def add_cluster(root: Root, cluster_name: str, config: IO[str]) -> None:
+    """
+    Create a new cluster and start its provisioning
+    """
+    if not root.quiet:
+        click.echo(f"Creating cluster {cluster_name}")
+    config_dict = yaml.safe_load(config)
+    await root.client._admin.add_cluster(cluster_name, config_dict)
+    if not root.quiet:
+        click.echo(f"Done")
```

Comment:
As currently, we don't poll cluster setup process maybe let us write something more verbose:
"Cluster successfully added and will be set up within 24 hours"

Reply:
And minor: not f-string

Reply:
The output is updated, thanks!